frontend/src/app/utils/text.ts

```diff
@@ -47,6 +47,14 @@ export function toLowerCaseHyphenate(string: string) {
   return string.toLocaleLowerCase().replace(/\s+/g, "-");
 }
 
+export function getSubStrAfterChar(
+  s: string,
+  strToSplitOn: string,
+  subStrLimit: number = 2
```

Comment:
for the `subStrLimit` arg, it looks like you are setting 2 as the default. I am wondering if you don't need to pass 2 in as an argument in the places you're calling it above?

Reply:
you are right -- I don't need to! I'll get rid of it where I call this function! 😸